telemetry/ui/src/components/routes/app/StepList.tsx

```diff
@@ -262,8 +262,10 @@ const LinkSubTable = (props: {
             <TableCell colSpan={1}>
               <Icon className={`h-5 w-5 ${iconColor} `} />
             </TableCell>
-            <TableCell colSpan={1} className={` ${normalText}  w-48 min-w-48 max-w-48 truncate`}>
-              <Link to={`/project/${props.projectId}/${subApp.child.app_id}`}>
+            <TableCell colSpan={1} className={` ${normalText} w-48 min-w-48 max-w-48 truncate`}>
+              <Link
+                to={`/project/${props.projectId}/${subApp.child.partition_key}/${subApp.child.app_id}`}
```

Comment:
Double-check that this can only be "null", if it's "undefined" it'll break.

Reply:
it seems to work fine with null

Reply:
i.e. the correct URL is created with the word "null"

Reply:
Yes just wanted to check it wasn't undefined.